class-pikpay.php

```diff
@@ -456,7 +456,11 @@ function process_payment($order_id)
         } else {
             //Form integration   
             return array('result' => 'success', 'redirect' => add_query_arg('order',
-                $order->get_id(), add_query_arg('key', $order->order_key, WC_Cart::get_checkout_url()))
+                //only works with php 7 
+                //$order->get_id(), add_query_arg('key', $order->order_key, WC_Cart::get_checkout_url()))
+
+                //php 8 version
+                $order->get_id(), add_query_arg('key', $order->order_key, wc_get_checkout_url()))
```

Comment:
Will this work for php version 7?

Reply:
@jasminsuljic It will work with php 7 - it's backward compatible, tested and production ready.